### DISCUSSION TITLE
How to remove packman files?

### DISCUSSION THREAD

**Discussion by MennoVink** [NVIDIA-Omniverse/PhysX#214] [Q&A] [answered]
I'm on windows and used the bat script to generate the project.
I noticed that this creates more files on my hdd than just the project. That would be fine if it would clean up those files after they're no longer needed, but it doesn't. I've now got a C:\packman-repo folder containing 908MB of files.

What would be the right way to remove these files, and are there more places where pollution might happen?
All i could find were unix commands but as mentioned i'm on windows and the pacman command is unrecognized.

**Comment by preist-nvidia** [comment, marked as answer]
Hi @MennoVink - Yes we don't have automated cleanup of the packman files. What I would suggest is to simply delete the whole packman repo folder and then rerun generate projects just for the configs that you are actively using to redownload the required dependencies.

**Comment by jcarius-nv** [comment]
Correction: The packman command has received a `prune` verb recently, so it's possible to do this in a little more controlled fashion than just deleting the entire folder:
```
~/git/physx/buildtools/packman$ ./packman prune --help
usage: packman prune [-h] [-v | -q | -s] [-d] DAYS_SINCE_LAST_ACCESS

positional arguments:
  DAYS_SINCE_LAST_ACCESS
                        number of days (can be fractional) that a pruned package must have remained unused

options:
  -h, --help            show this help message and exit
  -v, --verbose         increase output verbosity
  -q, --quiet           no output except for error messages
  -s, --silent          silence all output
  -d, --dry-run         provide a summary of actions that would be taken but take none
  ```

**Reply by preist-nvidia** [reply]
Thank you Jan - TIL!
